jest.config.js

```diff
@@ -1,11 +1,4 @@
+const baseConfig = require("./jest.config.base.js");
 module.exports = {
-  roots: ["<rootDir>/test"],
-  testMatch: ["**/*.test.ts"],
-  transform: {
-    "^.+\\.(j|t)sx?$": "ts-jest",
-    "^.+\\.svg$": "<rootDir>/fileTransformer.js",
-    "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$":
-      "<rootDir>/fileTransformer.js",
-  },
-  testEnvironment: "jest-environment-node",
+  ...baseConfig,
 };
```

Comment:
this change might not be needed if symlinks are used?

Reply:
👍